File: Forecast/app/src/main/java/gabyshev/denis/forecast/weather_api/weekPojo/WeekPojo.java

package gabyshev.denis.forecast.weather_api.weekPojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WeekPojo {

    @SerializedName("cod")
    @Expose
    private String cod;
    @SerializedName("message")
    @Expose
    private Double message;
    @SerializedName("cnt")
    @Expose
    private Integer cnt;
    @SerializedName("list")
    @Expose
    private java.util.List<gabyshev.denis.forecast.weather_api.weekPojo.List> list = null;
    @SerializedName("city")
    @Expose
    private City city;

    public String getCod() {
        return cod;
    }

    public void setCod(String cod) {
        this.cod = cod;
    }

    public Double getMessage() {
        return message;
    }

    public void setMessage(Double message) {
        this.message = message;
    }

    public Integer getCnt() {
        return cnt;
    }

    public void setCnt(Integer cnt) {
        this.cnt = cnt;
    }

    public java.util.List<gabyshev.denis.forecast.weather_api.weekPojo.List> getList() {
        return list;
    }

    public void setList(java.util.List<gabyshev.denis.forecast.weather_api.weekPojo.List> list) {
        this.list = list;
    }

    public City getCity() {
        return city;
    }

    public void setCity(City city) {
        this.city = city;
    }

}

File: Forecast/app/src/main/java/gabyshev/denis/forecast/weather_api/weekPojo/WeekPojo.kt
package gabyshev.denis.forecast.weather_api.weekPojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class WeekPojo {

    @SerializedName("city")
    @Expose
    var city: City? = null
    @SerializedName("cod")
    @Expose
    var cod: String? = null
    @SerializedName("message")
    @Expose
    var message: Double? = null
    @SerializedName("cnt")
    @Expose
    var cnt: Int? = null
    @SerializedName("list")
    @Expose
    var list: List<ListDay>? = null

}
